UT4MasterServer/Authentication/BasicAuthenticationHandler.cs

```diff
@@ -26,6 +27,11 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
 				return AuthenticateResult.Fail("unexpected scheme");
 			}
 
+			if (!authorization.Value.IsBase64())
+			{
+				return AuthenticateResult.Fail("not Base64");
+			}
+
```

Comment:
I would prefer if this was combined inside `ClientIdentification` class. it's constructor already tries to decode base64...

Reply:
Done